app/javascript/src/NewService/components/FormElements/SelectBackendApi.jsx

```diff
@@ -0,0 +1,38 @@
+// @flow
+
+import React, {useState} from 'react'
+import {Label} from 'NewService/components/FormElements'
+
+const SelectBackendApi = ({backendApis}: {backendApis: Array<{id: number, name: string}>}) => {
+  const [actAsProduct, setActAsProduct] = useState(false)
+
+  const toggleActAsProduct = (event) => {
+    setActAsProduct(event.target.checked)
```

Comment:
In React we should use `currentTarget` as it is safer.
```suggestion
    setActAsProduct(event.currentTarget.checked)
```